src/Umbraco.Web.UI.Client/src/packages/property-editors/label/property-editor-ui-label.element.ts

```diff
@@ -1,34 +1,40 @@
-import { html, customElement, property } from '@umbraco-cms/backoffice/external/lit';
-import { UmbTextStyles } from '@umbraco-cms/backoffice/style';
+import { customElement, html, property, state, when } from '@umbraco-cms/backoffice/external/lit';
+import { UmbLitElement } from '@umbraco-cms/backoffice/lit-element';
 import type {
 	UmbPropertyEditorUiElement,
 	UmbPropertyEditorConfigCollection,
 } from '@umbraco-cms/backoffice/property-editor';
-import { UmbLitElement } from '@umbraco-cms/backoffice/lit-element';
 
 /**
  * @element umb-property-editor-ui-label
  */
 @customElement('umb-property-editor-ui-label')
 export class UmbPropertyEditorUILabelElement extends UmbLitElement implements UmbPropertyEditorUiElement {
-	@property()
-	value = '';
+	@state()
+	private _labelTemplate?: string;
 
 	@property()
-	description = '';
+	value = '';
 
 	@property({ attribute: false })
-	public config?: UmbPropertyEditorConfigCollection;
+	public set config(config: UmbPropertyEditorConfigCollection | undefined) {
+		this._labelTemplate = config?.getValueByAlias('labelTemplate');
+	}
 
 	override render() {
-		return html`${this.value ?? ''}`;
+		return when(
+			this._labelTemplate?.length,
+			() => html`<umb-ufm-render inline .markdown=${this._labelTemplate} .value=${this.value}></umb-ufm-render>`,
+			() => html`${this.value ?? ''}`,
+		);
 	}
-
-	static override styles = [UmbTextStyles];
 }
 
+/** @deprecated Should be exported as `element` only; to be removed in Umbraco 18. */
 export default UmbPropertyEditorUILabelElement;
 
+export { UmbPropertyEditorUILabelElement as element };
```

Comment:
We ran into a problem during the hackathon at Codegarden, where we could not export these elements through an index.ts file, because several of them had the same export name (`"element"`). Is this safe enough to do all over the codebase?

Reply:
I'd done this (and added the deprecation notice) to align with how we've done the other property-editor element exports. I think our assumption was that the property-editor elements should be loaded via its manifest, but I can see that this could be problematic for subclassing/extending core property-editors.

I'm happy to make the appropriate changes, we just need to align across the other property-editor elements.

Reply:
I think I see what's going on: In some places, we only have `import` statements for elements and their files (to register them in the browser). In other places, we have `export` statements, which fundamentally do the same (registers in the browser), but also export the classes. 
It so happens that some of these (not yours in particular per se) get picked up in an index barrel and cause export conflicts from time to time.

If we ever want to let someone extend on these classes - and get their typings - we cannot export as `"element"` as that would prevent us from bundling them up.

As you say, we need to align on this, but perhaps we can avoid it on this new element for starters?

Reply:
I agree that the export conflicts is an issue to resolve. 👍
But not having a named `element` export goes against what we have in the `loadManifestElement` function, where it is first checking for an exported `element` then falls back on `default`.
e.g. https://github.com/umbraco/Umbraco-CMS/blob/release-16.0.0/src/Umbraco.Web.UI.Client/src/libs/extension-api/functions/load-manifest-element.function.ts#L26

It's not a big overhead to fall back on `default`, but maybe our intended logic is at odds with the practicality of bundling?

That question is beyond the scope of this PR, so for now, I'm happy to remove the `element` export line (and deprecation).

Reply:
The `default` export is only relevant for direct imports from the file itself, so not anything we have ever had public anyway.

I think it is fine that we scan for an `element` export first before doing the default fallback, as that is something for package developers to consider. However, it does not work in our use case, but it should be fine to leave as-is.